styles.css

```diff
@@ -61,6 +76,78 @@
   display: none;
 }
 
+.dropdown-menu {
+  background-color: #3c3a39;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 7%;
+  right: 0;
+  margin: 0;
+}
+
+.hide {
+  display: none;
+}
+
+.dropdown-option {
+  color: #fff5e1;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  border-bottom: 1px solid #6f6c6b;
+  margin: 0 6% 6% 6%;
+  text-decoration: none;
+  padding-left: 4%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.dropdown-option a {
+  color: #fff5e1;
+}
+
+.mobile-menu {
+  display: flex;
+  justify-content: end;
+}
+
+.mobile-menu button {
+  width: 8px;
+  height: 22px;
+  background: transparent;
+  border: none;
+  color: #67798e;
+  margin-right: 38px;
+  margin-top: 26px;
+  font-size: 25px;
+}
+
+.dropdown-bottom {
+  background-color: #e5e5e5;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 7%;
+}
+
+.dropdown-divline {
+  width: 134px;
+  height: 5px;
+  bottom: 8px;
+  background: #0d151e;
+  border-radius: 10px;
+  margin: 8px;
+  align-self: center;
+  margin-right: auto;
+  margin-left: auto;
+}
```

Comment:
[OPTIONAL] Following CSS best Practices, it would be better to use shorthand properties. Instead of using margin-right, margin-left separately you can simply use just one line: `margin: 8px auto;`

Reply:
you're right, thank you for that